templates/kubernetes/install.html

```diff
@@ -2,338 +2,634 @@
+              <li class="p-list__item">Private clouds</li>
+              <li class="p-list__item">Public clouds</li>
+              <li class="p-list__item">Hybrid clouds</li>
+            </ul>
+          </td>
+          <td>
+            <ul class="p-list--ticked u-no-padding">
+              <li class="p-list__item">Configurable Kubernetes</li>
+              <li class="p-list__item">Medium to large clusters</li>
+            </ul>
+          </td>
+          <td class="u-align--center">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg",
+              alt="",
+              width="14",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </td>          
+          <td class="u-align--center">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg",
+              alt="",
+              width="14",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </td>        
+        </tr>
+        <tr>
+          <td>
+            <p>Cloud Hosted (i)</p>
+          </td>
+          <td>
+            <ul class="p-list--ticked u-no-padding">
+              <li class="p-list__item">Public clouds</li>
+            </ul>
+          </td>
+          <td>
+            <ul class="p-list--ticked u-no-padding">
+              <li class="p-list__item">Traffic peaks</li>
+            </ul>
+          </td>
+          <td class="u-align--center">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg",
+              alt="",
+              width="14",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </td>
+          <td>Provided by the clouds</td>
+        </tr>
+        <tr>
+          <td>Kubeadm</td>
+          <td>
+            <ul class="p-list--ticked u-no-padding">
+              <li class="p-list__item">Bare metal</li>
+              <li class="p-list__item">Private clouds</li>
+              <li class="p-list__item">Edge/Micro clouds</li>
+            </ul>
+          </td>
+          <td>
+            <ul class="p-list--ticked u-no-padding">
+              <li class="p-list__item">DIY Kubernetes</li>
+            </ul>
+          </td>
+          <td class="u-align--center">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg",
+              alt="",
+              width="14",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </td>
+          <td class="u-align--center">-</td>
+        </tr>
+      </tbody>
+    </table>
+    <p>
+      <a href="/kubernetes/contact-us?product=kubernetes-install">Looking for help running Kubernetes? Get in touch
+        &nbsp;&rsaquo;</a>
```

Comment:
The &nbsp; should be touching the last character of the link otherwise it will wrap. Defeating the purpose of the nbsp